samples/bluetooth/mesh/blob_transfer/README.rst

```diff
@@ -0,0 +1,310 @@
+.. _ble_mesh_blob_xfer:
+
+Bluetooth Mesh: Binary Large Object (BLOB) Transfer
```

Comment:
If you use existing `mesh_shell` test application, and enable BLOB srv and cli models there. Then you can evaluate the BLOB transfer like this:

On BLOB sender shell:
```
mesh init
mesh cdb create 1234
mesh cdb show
mesh cdb app-key-add 0 0 
mesh prov local 0 1
mesh models cfg appkey add 0 0
mesh models cfg model app-bind 1 0 0x1400
mesh models cfg model app-bind 1 0 0x1401
mesh prov auth-method none
mesh prov remote-adv dddd0000000000000000000000000000 0 5 10
mesh target dst 5
mesh models cfg appkey add 0 0
mesh models cfg model app-bind 5 0 0x1401
mesh models cfg model app-bind 5 0 0x1400
mesh models blob cli target 0x05
```

On BLOB receiver shell:
```
mesh models blob srv rx 0xac
```

Then, again BLOB sender shell:
```
mesh models blob cli tx 0xac 1024 12 30
```

Reply:
Okay. I'll merge and configure only the parts that are not in the `mesh_shell` test.